### ISSUE TITLE
Display the dice that the user selected in results

### ISSUE THREAD

**Issue by mikikiv** [hone1er/threes#2]
After finishing a round when playing solo, it feels like random dice are returned to the score screen. 
Instead, the user should see the dice that they selected before starting a new round

Ex: if the user selects 3, 3, 4, 1, 1, then after the game those exact numbers on dice should be returned to the screen before selecting `reset`

In a multiplayer game, you could display the winner's combination of dice.

**Comment by hone1er** [comment]
Good idea. I suppose i could add a state to the game for selected dice where it initializes as `selectedDice=[0,0,0,0,0]`. 

Then as dice are selected, in the same step that changes the die's index in `dieVisable=[true, true, true, true, true]` array from true to false, the corresponding index in the `selectedDice` array would also be changed to the number. Then reset it to `[0,0,0,0,0]` for the next players turn.
